src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2796,7 +2838,7 @@ private static Symbol AsMemberOfType(NamedTypeSymbol containingType, Symbol symb
             var symbolDefContainer = symbolDef.ContainingType;
             while (true)
             {
-                if (containingType.OriginalDefinition.Equals(symbolDefContainer, TypeCompareKind.ConsiderEverything))
+                if (containingType.OriginalDefinition.Equals(symbolDefContainer, TypeCompareKind.AllIgnoreOptions))
```

Comment:
>AllIgnoreOptions [](start = 97, length = 16)

I didn't understand that change. Do you remember which scenario this affected?

Reply:
`NullableReferenceTypesTests.TypeInference_DynamicDifferences_02` fails where `containingType` is `(object!, object?)` and `symbol` is `(dynamic, object).Item1`.

---
In reply to: [221403920](https://github.com/dotnet/roslyn/pull/29727#discussion_r221403920) [](ancestors = 221403920)